spec/abci/abci++_methods.md

```diff
@@ -874,25 +874,4 @@ enum VerifyStatus {
         * If `Status` is `ACCEPT`, the consensus algorithm will accept the vote as valid.
         * If `Status` is `REJECT`, the consensus algorithm will reject the vote as invalid.
 
-
-### CanonicalVoteExtension
```

Comment:
I think there is a reference to this section: [here](https://github.com/cometbft/cometbft/blob/334030969b43202f70cf17af21b066bcc336555a/spec/abci/abci%2B%2B_methods.md?plain=1#L520). So removing this section will leave that reference dangling.